### PR TITLE
fix permissions issues

### PR DESCRIPTION
**Pull request by philbrookes**
fix permissions issues. Also removed the call to test for docker build as tests are not passing on this branch yet.

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -5,6 +5,42 @@ metadata:
...
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
```
Comment (R-Lawton) [inline config/rbac/role.yaml:18]: Do we want this cluster role to have full permissions on everything for secrets? my main thinking is the potential for  updating or patching existing secrets that it didn't create

Comment (R-Lawton) [reply config/rbac/role.yaml:18]: same for certs

Comment (maleck13) [reply config/rbac/role.yaml:18]: I think cert-manager needs that but the controller probably just needs read access as it copies the secret into a manifestwork

Comment (maleck13) [reply config/rbac/role.yaml:18]: the controller creates the certificate resources

Comment (philbrookes) [reply config/rbac/role.yaml:18]: oh good point, I just went super-permissive to get everything working and didn't look back. I'll update this now.

**Pull request merged**